src/Form/index.tsx

```diff
@@ -1570,7 +1572,7 @@ export default function FormWithRouter({
         {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
         {/* @ts-ignore */}
         <Route path='/'>
-          <Form {...props} formName={formName} key={formName} />
+          <Form {...props} formName={formName} key={formName} uuid={uuidv4()} />
```

Comment:
what if FormWithRouter gets rerendered? wouldn't this generate a new uuid for the same form